src/pages/Thanks/ThanksPage.vue

```diff
@@ -188,7 +188,16 @@ export default {
 			// But it will not throw a server error.
 			this.receipt = data?.shop?.receipt;
 			this.isGuest = this.receipt && !data?.my?.userAccount;
-			this.thanksPageVersion = this.isGuest ? 'b' : 'a';
```

Comment:
This line seemed to be overwriting the change in the mounted block for some reason, in my testing result sometimes finished after the mounting code... kind of puzzling.

Reply:
result is also called everytime the watchQuery call has a new result, so it could be that

Reply:
@emuvente good call, I didn't think about that!